pkg/helm/watches/watches.go

```diff
@@ -28,12 +28,13 @@ import (
 // Watch defines options for configuring a watch for a Helm-based
 // custom resource.
 type Watch struct {
-	GroupVersionKind        schema.GroupVersionKind
-	ChartDir                string
+	GroupVersionKind        schema.GroupVersionKind `yaml:",inline"`
+	ChartDir                string                  `yaml:"chart"`
 	WatchDependentResources bool
 	OverrideValues          map[string]string
```

Comment:
Can we add the `yaml:""` tags for these other fields and remove the `yamlWatch` struct?